Keda.Cosmosdb.Scaler/src/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Net;

```

Comment:
I looks like the location of these files are wrong - `Keda.Cosmosdb.Scaler/src/Program.cs`, shouldn't it be `src/Keda.Cosmosdb.Scaler/Program.cs` instead?